cloud-data-protection/src/components/navMenu/navMenu.tsx

```diff
@@ -0,0 +1,118 @@
+import React, {Fragment} from 'react';
+import {useSelector} from "react-redux";
+import {selectAuthenticated, selectUser} from "features/userSlice";
+import {
+    Home,
+    AccountCircle,
+    DriveEta,
+    Person,
+    PersonAdd,
+    ExitToApp
+} from '@material-ui/icons';
+import {
+    AppBar, Divider,
+    Drawer,
+    ListItem,
+    ListItemIcon,
+    ListItemText,
+    Toolbar,
+} from "@material-ui/core";
+import {Link} from "react-router-dom";
+import NavMenuItem from "components/navMenu/navMenuItem";
+import UserRole from "entities/userRole";
+import './navMenu.css';
+
+const NavMenu = () => {
+    const authenticated = useSelector(selectAuthenticated);
+    const user = useSelector(selectUser);
+
+    const menuItems = (): JSX.Element[] => {
+        const links: NavMenuItem[] = [];
+
+        links.push({text: 'Home', route: '/', icon: <Home /> });
+
+        if (authenticated) {
+            userMenuItems().forEach(link => links.push(link));
+        }
+
+        return links.map(link => mapLink(link));
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
        return links.map((link) => mapLink(link));
```